frontend/src/components/SchemaEdit.vue

```diff
@@ -49,6 +49,7 @@ export default {
   name: "SchemaEdit",
   components: {EditAttributes, TextInput, Checkbox},
   props: ["schema"],
+  emits: ["update-schema"],
```

Comment:
Do we use this event somewhere?

Reply:
Yup https://github.com/SUSE/aimaas/pull/207/files#diff-04b105b5ae587de59f4bab07a7c490d96d52b13ef8785ef5f138172a22eb5b95R16

Reply:
Ah :bulb: My bad, I did not check for the CamelCase version. Would it make to use the same formatting here as in the other file?

Reply:
yes, it works and it will be less confusing :+1:

Reply:
Cool, thank you :slightly_smiling_face: